Figures.cpp

```diff
@@ -93,38 +94,38 @@ bool canAttack(const figType type, int cordX, int cordY, int targetX, int target
 
 
 //powinno się gdzieś dodać movePoints
-bool styleA(int targetX, int targetY) { //+
+bool Figure::styleA(int targetX, int targetY, Board &board) { //+
     int x = cordX;
     int y = cordY;
     int changedX = x - targetX;
     int changedY = y - targetY;
 
     if(changedX > movePoints || changedY > movePoints) return 0;
 
-    if( changeX < 0 && y == targetY){ // poziomo lewo
+    if( changedX < 0 && y == targetY){ // poziomo lewo
         for(int i = 1; i <= changedX; i++){
-            if(Board[x-i][y] != /*EmptyField*/) return 0;
+            if(!board[x - i][y]->isEmpty()) return 0;
         }
     }
     else if(changedX > 0 && y == targetY){ //poziomo prawo
         for(int i = 1; i <= changedX; i++){
-            if(Board[x+i][y] != /*EmptyField*/) return 0;
+            if(board[x+i][y]->isEmpty()) return 0;
```

Comment:
nie ma negacji ja wszędzie niżej 
I think you should use an
`if(**!**board[x+i][y]->isEmpty()) return 0;` element here instead.